support/src/main/java/bisq/support/moderator/ModeratorService.java

```diff
@@ -180,7 +180,7 @@ public CompletableFuture<SendMessageResult> contactUser(UserProfile userProfile,
 
                     if (channel.getChatMessages().isEmpty() && isReportingUser) {
                         return twoPartyPrivateChatChannelService.sendTextMessage(Res.get("authorizedRole.moderator.replyMsg"),
-                                citationMessage.map(msg -> new Citation(userProfile.getId(), msg)),
+                                citationMessage.map(msg -> new Citation(userProfile.getId(), msg, "")),
```

Comment:
As we do not use the chatMessageId here, it would be better to make chatMessageId optional to reflect that it is a valid case that it is not set.